File: PHP/Produk.php
<?php

class Produk {
    protected $kodeProduk;
    protected $namaProduk;
    protected $merek;
    protected $harga;
    protected $tahunProduksi;
    protected $fotoProduk;

    public function __construct($kodeProduk = "", $namaProduk = "", $merek = "", $harga = 0, $tahunProduksi = "", $fotoProduk = "") {
        $this->kodeProduk = $kodeProduk;
        $this->namaProduk = $namaProduk;
        $this->merek = $merek;
        $this->harga = $harga;
        $this->tahunProduksi = $tahunProduksi;
        $this->fotoProduk = $fotoProduk;
    }

    public function getHeaders() {
        return ["Kode Produk", "Nama Produk", "Merek", "Harga", "Tahun Produksi", "Foto Produk"];
    }

    public function getData() {
        return [
            $this->kodeProduk,
            $this->namaProduk,
            $this->merek,
            "Rp" . number_format($this->harga, 0, ',', '.'),
            $this->tahunProduksi,
            $this->fotoProduk
        ];
    }
}

File: CPP/Produk.cpp
// File: Produk.cpp

#include <iostream>
#include <string>

using namespace std;

class Produk {
private:
    string kodeProduk;
    string namaProduk;
    string merek;
    int harga;
    int tahunProduksi;
    string status;

public:
    Produk(string kode = "", string nama = "", string merek = "", int harga = 0, int tahun = 0, string status = "") {
        this->kodeProduk = kode;
        this->namaProduk = nama;
        this->merek = merek;
        this->harga = harga;
        this->tahunProduksi = tahun;
        this->status = status;
    }

    // Setter
    void setKodeProduk(string kode) { this->kodeProduk = kode; }
    void setNamaProduk(string nama) { this->namaProduk = nama; }
    void setMerek(string merek) { this->merek = merek; }
    void setHarga(int harga) { this->harga = harga; }
    void setTahunProduksi(int tahun) { this->tahunProduksi = tahun; }
    void setStatus(string status) { this->status = status; }

    // Getter
    string getKodeProduk() const { return this->kodeProduk; }
    string getNamaProduk() const { return this->namaProduk; }
    string getMerek() const { return this->merek; }
    int getHarga() const { return this->harga; }
    int getTahunProduksi() const { return this->tahunProduksi; }
    string getStatus() const { return this->status; }

    ~Produk() {}
};
